client/src/components/App.jsx

```diff
@@ -27,12 +31,43 @@ class App extends React.Component {
       url: 'https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfe/products',
       headers: {Authorization: Token},
       success: (data) => {
-        this.setState({allProducts: data});
-        this.setState({product: data[0]});
+        this.setState({allProducts: data})
+        this.getProductDetails(data[0].id)
+        this.getProductStyle(data[0].id)
+        this.updateProductId(data[0].id)
+      },
+      error: (err) => {
+        console.log(err);
       }
     });
   }
 
+  getProductDetails(id) {
+    $.get({
+      url: `https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfe/products/${id}`,
+      headers: {Authorization: Token},
+      success: (data) => {
+        this.setState({product: data})
+      },
+      error: (err) => {
+        console.log(err)
+      }
+    })
+  }
+
+  getProductStyle(id) {
+    $.get({
+      url: `https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfe/products/${id}/styles`,
+      headers: {Authorization: Token},
+      success: (data) => {
+        this.setState({productStyles: data})
+      },
+      error: (err) => {
+        console.log(err)
+      }
+    })
+  }
+
```

Comment:
in app.jsx when rendering your widget, you have to pass in styles
<Overview product={this.state.product} styles = {this.state.productStyles}/>

Reply:
`<Overview product={this.state.product} styles = {this.state.productStyles}/>`